Makefile

```diff
@@ -118,7 +112,7 @@ demo: generate lint manifests ## Run in demo mode
 	go run -ldflags $(LDFLAGS) ./main.go -namespace=$(RUN_NAMESPACE) -dev -demo-mode
 
 .PHONY: run-test-mode
-run-test-mode: generate fmt vet manifests ## Run against the configured Kubernetes cluster in ~/.kube/config
+run-test-mode: generate fmt-check lint manifests ## Run against the configured Kubernetes cluster in ~/.kube/config
```

Comment:
I think this was using `fmt` instead of `fmt-check` so that code would be fixed before being passed to the compiler.

Reply:
I think fmt-check is correct as it's using the gofmt.sh script under /hack
fmt was just an alias for lint
vet is also an alias, but we will remove it later to not break the current jobs